lld/ELF/Arch/LoongArch.cpp

```diff
@@ -22,6 +22,40 @@ using namespace lld;
 using namespace lld::elf;
 
 namespace {
+#define LARCH_GET_RD(insn) (insn & 0x1f)
+#define LARCH_GET_RJ(insn) ((insn >> 5) & 0x1f)
+#define LARCH_MK_ADDI_D 0xffc00000
+#define LARCH_OP_ADDI_D 0x02c00000
```

Comment:
> Can we reuse the values in below enum (i.e. line 93)?

Actually, I'm thinking if we could remove the enum values of Op and use the newly defined macros for all operations about instruction opcode? Because we may need to use the mask of instruction sometimes.